core/watcher/eventsource/namespace.go

```diff
@@ -26,44 +26,62 @@ type NamespaceQuery Query[[]string]
 type NamespaceWatcher struct {
 	*BaseWatcher
 
-	out chan []string
-
 	// TODO (manadart 2023-05-24): Consider making this plural (composite key)
 	// if/when it is supported by the change log table structure and stream.
```

Comment:
I don't think this TODO is still relevant

Reply:
It is if you want to support multiple initial queries.